### DISCUSSION TITLE
Renaming `-ms` to `-airdrops` OR merging the V2 Core and MS subgraphs into one

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/indexers#53] [Ideas]
Fjord Foundry has recently interacted with the Sablier subgraphs, and there have been several cases of confusion caused by having two different subgraphs for Lockup and Airstreams:

<img width="350" alt="SCR-20241127-kreo" src="https://github.com/user-attachments/assets/c1a7a774-1445-451c-a925-89aa610a6e43">
<img width="350" alt="SCR-20241127-krgf" src="https://github.com/user-attachments/assets/7f24e0fb-33cf-4fda-8326-e8b2f038c0ac">

We would avoid these issues altogether if we either (i) renamed `-ms` to `-airdrops`, for clarity OR (ii) merged the two subgraphs into one.

As discussed elsewhere (Slack?), to avoid backward-compatibility issues, we can just leave the `-ms` subgraphs as is and recreate other `-airdrops` subgraphs. This refactor would also be in line with the recent package tethering-related changes in Lockup.

cc @sablier-labs/frontend for feedback

**Comment by PaulRBerg** [comment]
With the launch of Flow and package tethering versioning, the name of the subgraphs have become increasingly confusing and misleading for integrators.

I imagine this is a chore, but I think it would be very helpful to create new subgraphs `sablier-lockup-*` to supersede `sablier-v2-*` subgraph series.

**Reply by razgraf** [reply]
I agree it'd make things clearer but it's a chore that will take some time to set up so I'd leave it for next year, with the added note that we can re-prioritize it if integrators start sharing this concern with us.

**Reply by PaulRBerg** [reply]
Understood.

I think I have recently seen a question about subgraphs on Telegram, but I can't remember where exactly.

It is worth noting that the longer we prolong this refactor, the more difficult it will be to do it when we eventually do it because we keep expanding to more and more EVM chains:

https://github.com/sablier-labs/docs/pull/216/files

**Comment by PaulRBerg** [comment]
Closing, tracking in https://github.com/sablier-labs/subgraphs/issues/57.
